deploy/test-environments/fleet_api/src/api/common_api.py

```diff
@@ -142,3 +142,111 @@ def get_build_info(version: str, is_snapshot: bool) -> str:
             f"API call failed, status code {api_ex.status_code}. Response: {api_ex.response_text}",
         )
         return ""
+
+
+def get_stack_latest_version():
+    """
+    Retrieve the latest version of the stack from the Elastic snapshots API.
+
+    Returns:
+        str: The latest version of the stack.
+
+    Raises:
+        APICallException: If the API call to retrieve the version fails.
+
+    """
+    url = "https://snapshots.elastic.co/latest/master.json"
+    try:
+        response = perform_api_call(
+            method="GET",
+            url=url,
+        )
+        response_obj = munchify(response)
+        return response_obj.version
```

Comment:
I know you really like munchify, but it seems like an overkill to transform the object just for one property access.

Reply:
removed munchify and updated return value